Lib/fontbakery/profiles/microsoft.py

```diff
@@ -1,4 +1,8 @@
 PROFILE = {
+    "include_profiles": ["opentype"],
+    "pending_review": [
+        "opentype:cff_ascii_strings",
+    ],
     "sections": {
```

Comment:
This profile was intentionally not importing other profiles, because MS wants to review any checks enabled by this profile and nit have fonts suddenly fail validation when FB is updated because a new check was added to one of the imported profiles.